src/__internal__/utils/helpers/events/events.ts

```diff
@@ -0,0 +1,256 @@
+import composedPath from "./composedPath";
+
+/**
+ * JavaScript Events
+ *
+ * A collection of functions to be used with events.
+ *
+ * E.g. The KeyPress event has different implementations accross browsers, so
+ * this class contains methods to polyfill this functionality to ensure a standardised
+ * implementation between browsers
+ *
+ */
+const Events = {
+  /**
+   * A method to determine if an event is of a particular type
+   * */
+  isEventType: (ev: Event, type: string): boolean => {
+    return ev.type === type;
+  },
+
+  /**
+   * A method to determine whether a key down event was an arrow key
+   * */
+  isNavigationKeyup: (ev: KeyboardEvent): boolean => {
```

Comment:
```suggestion
  isNavigationKeyup: (ev: KeyboardEvent | React.KeyboardEvent): boolean => {
```

I've realised from using these that they need to be able to handle React's synthetic events as well as native browser ones!